src/oscar/templates/oscar/partials/nav_primary.html

```diff
@@ -42,7 +42,7 @@
                         </a>
                         <ul class="dropdown-menu" data-navigation="dropdown-menu">
                             {% category_tree depth=2 as tree_categories %}
-                            <li><a tabindex="-1" href="{% url 'catalogue:index' %}">{% trans "All products" %}</a></li>
+                            <li><a tabindex="-1" href="{{ OSCAR_HOMEPAGE }}">{% trans "All products" %}</a></li>
```

Comment:
This also needs to stay as `'catalogue:index'` as we don't know whether the home page is the catalogue root.

Reply:
done!